azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/resources/bindings.json

```diff
@@ -526,6 +526,33 @@
           "help": "$sql_input_connection_help"
         }
       ]
+    },
+    {
+      "type": "daprBindingTrigger",
+      "displayName": "$daprBindingTrigger_displayName",
+      "direction": "trigger",
+      "enabledInTryMode": true,
+      "settings": [
+        
```

Comment:
we may add some setting parameters here, so that `functions:add` could prompt users to set the required values. For example, `pubSubName` and `pubSubName`

Reply:
These are hardcoded in the function itself because same name needs to be configured in component's yaml file. Thats the reason it is not added as a prompt.